Kickstarter-iOS/DataSources/HelpDataSource.swift

```diff
@@ -4,7 +4,7 @@ import KsApi
 final class HelpDataSource: ValueCellDataSource {
   func configureRows() {
     _ = HelpSectionType.allCases.map { section -> Void in
-      let values = section.cellRowsForSection.map { SettingsCellValue(user: nil, cellType: $0) }
+      let values = section.cellRowsForSection.map { SettingsCellValue(cellType: $0) }
```

Comment:
If you want to be point-free:
```suggestion
      let values = section.cellRowsForSection.map(SettingsCellValue(cellType:))
```